hangman.py

```diff
@@ -0,0 +1,192 @@
+              get_guessed_word(secret_word, letters_guessed))
+    return warnings, attemps
+
+
+def check_let(attemps, letter, letters_guessed):
+    '''
+    check_vowels_let: checks for the content of the vowel in the word
+        if the letter is correctly guessed the issuance of the word with it
+    attemps: variables whose have static number
+    letter: the letter entered by the player
+    returns: number of attemps and letter entered by the player
+    '''
+    # check letters in word
+    splited_word = get_guessed_word(secret_word, letters_guessed)
+    if letter not in secret_word:
+        print('Oops! That letter is not in my word:', splited_word)
+        # If letters was vowels remove 2 attemps, if not remove 1 attemps
+        if letter in VOWELS:
+            attemps -= 1
+        attemps -= 1
+    else:
+        print('Good guess:', splited_word)
+    return attemps, letter, letters_guessed
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+        secret_word: string, the word the user is guessing
+        letters_guessed: list (of letters), which letters have been guessed so far
+        returns: string, comprised of letters, underscores (_), and spaces that represents
+          which letters in secret_word have been guessed so far.
+    '''
+    splited_word = list(secret_word)
+    # Make from word string what consist lower space(UNKNOWN_LETTER)
+    for i in range(len(secret_word)):
+        if secret_word[i] not in letters_guessed:
+            splited_word[i] = UNKNOWN_LETTER + " "
+    return "".join(splited_word)
+
+
+def get_available_letters(letters_guessed):
+    '''
+        letters_guessed: list (of letters), which letters have been guessed so far
+        returns: string (of letters), comprised of letters that represents which letters have not
+          yet been guessed.
+    '''
+    avaible_letters = string.ascii_lowercase
+    # Find letters have been entered and deleted him from available_letters
+    alphabet = filter(lambda letter: letter not in letters_guessed, string.ascii_lowercase)
+    return ''.join(alphabet)
+
+
+def hangman(secret_word):
+    '''
+        secret_word: string, the secret word to guess.
+
+        Starts up an interactive game of Hangman.
+    '''
+    # constant numbers
+    warnings = 3
+    attemps = 6
+    letters_guessed = set()
+    # Message with first information and input letters
+    print(f'I am thinking of a word that is {len(secret_word)} letters long')
+    print(f'Your have {warnings} warnings')
+    while attemps > 0 and not is_word_guessed(secret_word, letters_guessed):
+        print("-" * 13)
+        print(f'You have  {attemps} guesses left.')
+        print("Available letters: ", get_available_letters(letters_guessed))
+        letter = input("enter letter: ").strip(' ').lower()
+        # Check letters and make a solve right letter or not
+        if letter == HINTS:
```

Comment:
You just removed implementation of hangman without hints. Try add to input arguments of function `hints_enabled`, and depending on this argument - check this hint